eng/Versions.props

```diff
@@ -283,12 +284,12 @@
       create a test insertion in Visual Studio to validate.
     -->
     <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
-    <StreamJsonRpcVersion>2.12.7-alpha</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>2.12.25</StreamJsonRpcVersion>
     <!--
       When updating the S.C.I or S.R.M version please let the MSBuild team know in advance so they
       can update to the same version. Version changes require a VS test insertion for validation.
     -->
-    <SystemCollectionsImmutableVersion>5.0.0</SystemCollectionsImmutableVersion>
+    <SystemCollectionsImmutableVersion>6.0.0</SystemCollectionsImmutableVersion>
```

Comment:
MSBuild and VS are both on 6.0.0 S.C.I so we are likely good here.

Reply:
They are also on 6.0.0 S.R.M in case that is a change we want to make at this time.